modules/light-clients/08-wasm/keeper/msg_server_test.go

```diff
@@ -219,7 +220,51 @@ func (suite *KeeperTestSuite) TestMsgMigrateContract() {
 					return &wasmvmtypes.Response{Data: data}, wasmtesting.DefaultGasUsed, nil
 				}
 			},
-			clienttypes.ErrInvalidClient,
+			types.ErrWasmInvalidContractModification,
+		},
+		{
```

Comment:
Nit: I would add these tests to `TestWasmMigrate` so that we test these things at the same level as done in `TestWasmSudo`, but here it's also ok.